pkg/api/short_url.go

```diff
@@ -57,6 +57,11 @@ func (hs *HTTPServer) redirectFromShortURL(c *models.ReqContext) {
 		return
 	}
 
+	// Failure to update last seen at should still allow to redirect
+	if err := hs.ShortURLService.UpdateLastSeenAt(c.Req.Context(), shortURL); err != nil {
```

Comment:
Might be a nice candidate for an async function but probably overkill for v1 🤷

Reply:
Yeah probably, but good idea :smile: